kubernetes/loculus/values.yaml

```diff
@@ -26,7 +26,12 @@ dataUseTerms:
     open: https://#TODO-MVP/open
     restricted: https://#TODO-MVP/restricted
 s3:
-  enabled: false
+  enabled: true
+  bucket:
+    endpoint: hel1.your-objectstorage.com
+    bucket: loculus-preview-private
+    accessKey: "[[s3accessKey]]"
+    secretKey: "[[s3secretKey]]"
```

Comment:
Can we move this to the preview values? For the default values.yaml, we should disable it.

Reply:
https://github.com/loculus-project/loculus/pull/3924/commits/603703496178b0dd7879e12469e0e1c717b61427